manim/mobject/coordinate_systems.py

```diff
@@ -814,6 +814,29 @@ def construct(self):
                     f"x={x} not located in the range of the graph ([{self.p2c(graph.get_start())[0]}, {self.p2c(graph.get_end())[0]}])",
                 )
 
+    def input_to_graph_coords(self, x: float, graph: "ParametricFunction") -> Tuple:
+        """
+        Returns a tuple of the axis relative coordinates of the point
+        on the graph based on the x-value given.
+
+        Examples
+        --------
+
+        .. code-block:: python
+
+            ax = Axes()
+            curve = ax.get_graph(lambda x: x ** 2)
+            ax.input_to_graph_coords(x=3, graph=curve)
+            # (3, 9)
```

Comment:
Let's turn this into a proper doctest to improve test coverage. Also `get_graph` -> `plot`.

```suggestion
            >>> from manim import Axes
            >>> ax = Axes()
            >>> parabola = ax.plot(lambda x: x ** 2)
            >>> ax.input_to_graph_coords(x=3, graph=parabola)
            (3, 9)
```